INSTALL.md

```diff
@@ -1,25 +1,37 @@
 # Installing
 
+## Apple macOS
+Install via [Homebrew Cask](https://brew.sh/) or download a pre-built app bundle for macOS (10.13+) from [Releases](https://github.com/borgbase/vorta/releases).
+```
+$ brew cask install vorta
+```
 
-## Install from Source
-The generic way is to install it as Python package using [PIP](https://pip.readthedocs.io/en/stable/installing/). First [install](https://borgbackup.readthedocs.io/en/stable/installation.html) Borg using the package of your distribution or via PyPI. The latter needs some additional [source packages](https://borgbackup.readthedocs.io/en/stable/installation.html#dependencies). Then install Vorta from PyPI. Your local Python version should be >= 3.6.
+## Linux
+
+### Flatpak
+
+<a href='https://flathub.org/apps/details/com.borgbase.Vorta'><img width='240' alt='Download on Flathub' src='https://flathub.org/assets/badges/flathub-badge-en.png'/></a>
+
+Settings can be transfered by copying ~/.local/share/Vorta/settings.db to ~/.var/app/com.borgbase.Vorta/data/Vorta/.
```

Comment:
transferred

Reply:
Also, the standard way should be the distribution package and the flatpak should only be the second best option.

For dist packages, you automatically get security updates and also they are usually cryptographically signed. Also they are a natural part of your linux installation, like all the other packages, not something extra / different.

flatpak could be recommended if a newer version is desired than provided by dist packages.

Reply:
> Also, the standard way should be the distribution package and the flatpak should only be the second best option.
> 
> For dist packages, you automatically get security updates and also they are usually cryptographically signed. Also they are a natural part of your linux installation, like all the other packages, not something extra / different.
> 
> flatpak could be recommended if a newer version is desired than provided by dist packages.

While I don't agree that Flatpak should come after distribution packages, I think we can postpone this to the time the first distribution package is published :)

Reply:
> transferred

Changed :)